02_activities/assignments/assignment.sh

```diff
@@ -1,22 +1,22 @@
 #!/bin/bash
  set -x
-m
+
 ################
 # Instructions #
 ################
 # > Edit this file below to complete the homework assignment
 # > Add your code below each comment to complete the tasks
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
- mkdir dir1, mkdir dir2, mkdir dir3, mkdir dir4, mkdir dir5
+ mkdir dir1 ... dir5
 # 2. List the contents of the parent directory to verify the presence of the 5 directories
 ls
 # 3. Create 5 text files in dir2 named file1, file2, file3, file4, and file5
-$ touch dir2/file1.txt
-$ touch dir2/file2.txt
-$ touch dir2/file3.txt
-$ touch dir2/file4.txt
-$ touch dir2/file5.txt
+touch dir2/file{1..5}.txt
```

Comment:
If you are creating all five files here, then there are no need to create them again in the following lines